content/docs/building-apps/setup-custodial-account.mdx

```diff
@@ -0,0 +1,189 @@
+ *
+ * @param  {string}   from   The public key of the Stellar account that
+ *     submitted this deposit.
+ * @param  {any}      to     A string representing the customer that's part of
+ *     this pooled account. This could be a key, a database index, an object,
+ *     or some other abstraction. The main point is that this is *NOT* a
+ *     Stellar account. This value either directly comes from or somehow
+ *     resolves from the transaction memo field on the payment.
+ * @param  {Asset}    asset  The asset deposited into the customer's account.
+ * @param  {string}   amount The quantity (as a string, conforming to the way
+ *     payments are submitted on the Stellar network) of `asset` being
+ *     debited.
+ **/
+function depositIntoPool(from, to, asset, amount) {}
+
+/**
+ * Withdraws an `amount` of `asset` from a customer (`from`).
+ *
+ * For simplicity, we assume that this function always works. However, you
+ * should obviously be managing balances appropriately and ensuring that
+ * a particular customer can in fact fulfill a requested withdrawal.
+ *
+ * Note that this should probably be executed *after* the requisite transaction
+ * (see `createPayment()`) succeeds to ensure accounts aren't debited
+ * unnecessarily.
+ *
+ * @param  {any}    from    As in `depositIntoPool()`, this is some
+ *     representation of a customer that's part of a pooled account but not
+ *     on the Stellar network.
+ * @param  {Asset}  asset   The asset being credited from the customer.
+ * @param  {string} amount  The quantity of `asset` to withdraw.
+ **/
+function withdrawFromPool(from, asset, amount) {}
+
+/**
+ * Creates a PaymentOp corresponding to a withdrawal.
+ *
+ * This is just a convenience method for submitting payments on behalf of
+ * customers in a transaction. You may want to just do one operation per
+ * transaction (for example to include the source customer's ID in the memo),
+ * or you can batch many operations together; it's up to you.
+ *
+ * @param  {string} to      The Stellar account address of the recipient
+ * @param  {Asset}  asset   The asset to credit from the source customer
+ * @param  {string} amount  The amount of `asset` to credit
+ *
+ * @return {xdr.paymentOp}
+ */
+function createPayment(to, asset, amount) {
+  return sdk.Operation.payment({
+    source: custodialAcc.accountId(),
+    destination: to,
+    asset: asset,
+    amount: amount,
+  });
+}
+```
+
+</CodeExample>
+
+
+## Integration points for listing XLM
+
+### Setting the memo required flag on your account
+Before we get into the main integration points for adding XLM, make sure to configure your pooled account to require memos. This step is necessary to prevent users from forgetting to attach a memo, which can result in additional support needed from the custodial services and a less desirable user experience for new customers. [This article](https://www.stellar.org/developers-blog/fixing-memo-less-payments) explains how to set that up.
```

Comment:
```suggestion
Before we get into the main integration points for adding XLM support to your product or service, make sure to configure your pooled account to require memos. This step is necessary to prevent users from forgetting to attach a memo, which can increase customer support requests and a lead to a less desirable user experience for new customers. [This article](https://www.stellar.org/developers-blog/fixing-memo-less-payments) explains how to set that up.
```